config/katello.migrations/160425017684-default-http.rb

```diff
@@ -0,0 +1,3 @@
+answers['foreman_proxy']['http'] = false if answers['foreman_proxy'].is_a? Hash
+
+answers['capsule'].delete('foreman_proxy_http') if answers['capsule'].is_a? Hash
```

Comment:
I'm still confused why this is here, before it was setting the value in foreman_proxy from the old capsule value (which was already done: https://github.com/Katello/katello-installer/blob/master/config/katello.migrations/160120212701-foreman-proxy.rb#L67), but now its just deleting it.  That's not required either, Kafo will do this since the param doesn't exist anymore.